src/posix/main.c

```diff
@@ -392,7 +385,7 @@ int main(int argc, char *argv[])
 #endif
 #endif
 
-    while (true)
+    while (!otSysPseudoResetWasRequested())
```

Comment:
can we make otSysMainloopPoll() return `-2` to indicate requesting pseudo-reset? Thus we don't need a new API.

Reply:
ok